yearn/.eslintrc

```
@@ -1,15 +1,15 @@
 {
-    "extends": ["eslint-config-keep", "prettier"],
-    "parserOptions": {
-      "ecmaVersion": 2017,
-      "sourceType": "module"
-    },
-    "env": {
-      "es6": true,
-      "mocha": true
-    },
-    "rules": {
-      "new-cap": "off"
-    }
+  "extends": ["eslint-config-keep", "prettier"],
```

`prettier` is already included in `eslint-config-keep`.
```suggestion
  "extends": ["eslint-config-keep"],
```

---

See https://github.com/keep-network/tbtc-v2/pull/14/commits/52313e312cf5491832a64b83bef5c4db38432d0d